.github/workflows/pr-test.yml

```diff
@@ -0,0 +1,24 @@
+name: PR-Test
```

Comment:
please include an action with the [golangci/golangci-lint](https://github.com/golangci/golangci-lint) linter.

Reply:
added